src/axolotl/utils/trainer.py

```diff
@@ -719,6 +720,10 @@ def setup_trainer(cfg, train_dataset, eval_dataset, model, tokenizer, total_num_
         **trainer_kwargs,
     )
 
+    if cfg.use_wandb and cfg.eval_table_size > 0:
```

Comment:
```suggestion
    if cfg.log_predictions_to_wandb and cfg.eval_table_size > 0:
```

Reply:
`cfg.use_wandb` is used to ensure WandB is actually enabled / available -- it is not new. Otherwise it will fail and say WandB is not initialized.

The `cfg.eval_table_size > 0` is the real way this is enabled, so `cfg.log_predictions_to_wandb`  would be redundant.

Do you agree?

Reply:
Discussed and agreed on Discord: https://discord.com/channels/1104757954588196865/1149562976429424750/1151303256794996847
Closing.